scripts/rsfcreator.py

```diff
@@ -77,14 +75,20 @@ def rsf_for_line(line_dict, key_field, entry_type, key_prefix='', key=None):
             entry_type, key_prefix, key, timestamp, item_hash)
     return (item_line, entry_line)
 
+def merge_level_2(dct, other):
+    for k, v in other.items():
+        if (other.__contains__(k)):
```

Comment:
The use of underscored methods should be avoided as much as possible. A common way to check the same would be:

```python
if k in other:
  dct[k] ...
```